assets/app/scripts/services/data.js

```diff
@@ -108,10 +92,6 @@ angular.module('openshiftConsole')
     $rootScope.$on( "$routeChangeStart", function(event, next, current) {
       self._websocketEventsMap = {};
     });
-
-    this.oApiVersion = "v1";
-    this.k8sApiVersion = "v1";
```

Comment:
thats fine, i think the server version was the important info there anyway and what we had the issue open for